pkg/controllers/leaderworkerset_controller.go

```diff
@@ -105,6 +105,9 @@ func (r *LeaderWorkerSetReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		log.Error(err, "Fetching leader statefulset")
 		return ctrl.Result{}, err
 	}
+	if leaderSts == nil {
+		r.Record.Eventf(lws, corev1.EventTypeNormal, "GroupsAreProgressing", fmt.Sprintf("Creating leader sts %s", lws.Name))
+	}
```

Comment:
probably better to have this right before calling `SSAWithStatefulset` which actually does the creation.